recorder.go

```diff
@@ -7,8 +7,8 @@ import (
 	"strconv"
 	"time"
 
-	trace "cloud.google.com/go/trace/apiv1"
 	"github.com/golang/protobuf/ptypes/timestamp"
+	gax "github.com/googleapis/gax-go"
```

Comment:
One concern about `gax` is : **This project is currently experimental and not supported**